TSOClient/tso.client/UI/Panels/EODs/UIPizzaMakerEOD.cs

```diff
@@ -187,7 +187,7 @@ public void P_Result(string evt, string txt)
             int resultVal = 1;
             int.TryParse(txt, out resultVal);
             if (resultVal > 1 && resultVal < 5) SetTip(Script.GetString("strResultSuccess"));
-            else if (resultVal > 4 && resultVal < 8) SetTip(Script.GetString("strResultSuccessBonus"));
+            else if (resultVal > 5 && resultVal < 8) SetTip(Script.GetString("strResultSuccessBonus"));
```

Comment:
This change incorrectly excludes `5` from the success range. There was nothing to change since the ranges did not intersect.